src/MediaPlayer/MediaPlayer.cs

```diff
@@ -18,6 +18,7 @@ namespace XamCore.MediaPlayer {
 	[Native]
 	[NoMac]
 	[NoTV]
+	[Availability (Deprecated = Platform.iOS_9_0)]
```

Comment:
Minor: in the headers I see the same deprecated messages for all these enums:  `Use AVPlayerViewController in AVKit.`. I think it's worth updating them all :P